src/modules/flow/file/file.json

```diff
@@ -89,6 +89,12 @@
           },
           {
             "data_type": "int",
+            "default": {
+              "max": "INT32_MAX",
+              "min": "INT32_MIN",
+              "step": 1,
+              "val": 644
```

Comment:
Is this value correct? `644` makes sense if it's in octal, but this is in decimal. Use `420` instead.